pkg/app/app.go

```diff
@@ -104,6 +105,27 @@ func Init(app *App) *App {
 		app.Logger.Info("Live output is enabled")
 	}
 
+	if app.FileOrDir != "" {
+		absPath, err := app.fs.Abs(app.FileOrDir)
+		if err != nil {
+			panic(err)
+		}
+		fileStat, err := app.fs.Stat(absPath)
+		if err != nil {
+			panic(err)
+		}
+		if fileStat.IsDir() {
+			app.RootDir = absPath
+		} else {
+			app.RootDir = app.fs.Dir(absPath)
+		}
+	} else {
+		app.RootDir, err = app.fs.Getwd()
```

Comment:
This might result in nested helmfile.yaml files refer to tpl files in unexpected directories!

Let's say you have the following tree:

- /path/to
  - helmfile.yaml
  - _helpers.tpl 
  - subhelmfiles/
    - helmfile.yaml
    - _helpers.tpl

This app.RootDir always point to `/path/to` even if helmfile is currently processing `/path/to/subhelmfiles/helmfile.yaml`?

Ideally helmfile's include would work:

- `/path/to/helmfile.yaml` implicitly loads `/path/to/_helpers.tpl` and
- `/path/to/subhelmfiles/helmfile.yaml` implicitly loads `/path/to/subhelmfiles/_helpers.tpl`

However I think this would result in:


- `/path/to/helmfile.yaml` implicitly loads `/path/to/_helpers.tpl` and
- `/path/to/subhelmfiles/helmfile.yaml` implicitly loads `/path/to/_helpers.tpl`(loading the helpers in the root! this breaks helmfile's design rule that each (sub-)helmfile is independently runnable.